CMakeLists.txt

```diff
@@ -479,6 +479,11 @@ include_directories(SYSTEM ${PROJECT_BINARY_DIR}/include)
 
 if (NOT WITH_SYSTEM_BOOST)
   LIST(APPEND Boost_LIBRARIES "-lz")
+else()
```

Comment:
@tchaikov 
Sure.

As for your other remark: I guess boost libs are used quite a lot.
But the Python part of it, only in ceph-mgr I think, since I ran into it when enabling WITH_MGR.